java/src/main/native/CMakeLists.txt

```diff
@@ -146,11 +151,13 @@ endif (CUDF_INCLUDE AND CUDF_LIBRARY)
 
 find_path(RMM_INCLUDE "rmm"
           HINTS "$ENV{RMM_ROOT}/include"
+                "$ENV{RMM_HOME}/include"
                 "$ENV{CONDA_PREFIX}/include/rmm"
                 "$ENV{CONDA_PREFIX}/include")
 
 find_library(RMM_LIBRARY "rmm"
              HINTS "$ENV{RMM_ROOT}/lib"
```

Comment:
Is this necessary? If others are OK I would like to remove this

Reply:
In the RMM case, this is already planned to be removed by #5632.  Let's keep this PR focused on the necessary changes.